samples/IISSample/web.config

```diff
@@ -1,9 +0,0 @@
-﻿<?xml version="1.0"?>
```

Comment:
Should we remove publish-iis tool then? Because then it will go and modify the web.config so what you show is not what you get...